pkg/controller/nodedaemon/daemonsets.go

```diff
@@ -91,6 +93,43 @@ func getDiskMakerDSMutateFn(
 	}
 }
 
+func getLVDiskMakerDSMutateFn(
```

Comment:
we don't need a separate daemonset for LocalVolumes

Reply:
How would the `DaemonSet` have both `localVolume, LocalVolumeSet` info?